### DISCUSSION TITLE
Adding request parameters in Url

### DISCUSSION THREAD

**Discussion by DharaVaghasiya** [mockoon/mockoon#550] [Help] [answered]
I'm trying to mock api with url :
/{student}/{institutionId}/admission/Payment/deposit

where,
student : string and institutionId : string are two parameters passed in api request 
API : PUT

I'm not able to find way to mock this API.
Please help to mock this url
Thank you

**Comment by 255kb** [comment, marked as answer]
Mockoon is using Expressjs and do not support url parameters with `{ }`. The correct syntax is the following: 
`/:student/:institutionId/admission/Payment/deposit`. 
URL parameters must be prefixed with `:`.

More on this here: https://expressjs.com/en/guide/routing.html#route-parameters

I will open an issue to add more info on this in the documentation.

**Reply by DharaVaghasiya** [reply]
Thank you for reply
